src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,57 @@
-export const PeopleFilters = () => {
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { useCallback } from 'react';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+const startedCenturies = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+
+  const handlerCenturiesChange = useCallback((el: string) => {
+    if (centuries.includes(el)) {
+      return { centuries: [...centuries].filter(cent => cent !== el) || [] };
```

Comment:
don't see any sense to make a copy of centuries, because method filter did it already 
```suggestion
      return { centuries: centuries.filter(cent => cent !== el) || [] };
```